ui/app/src/components/ContentTypeManagement/components/TypeBuilderFormsEngine.tsx

```
@@ -0,0 +1,279 @@
+		</ErrorBoundary>
+	);
+}
+
+function FieldBreadcrumbs(props: FieldFormViewProps): JSX.Element {
+	if (!props.field) return;
+	const fieldPathIds = props.fieldIdPath?.split('.') ?? [];
+	return (
+		fieldPathIds.length > 1 && (
+			<Breadcrumbs separator={<NavigateNextIcon fontSize="small" />}>
+				{fieldPathIds.map((id) => (
+					<Typography variant="body2">{id}</Typography>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)